src/js/htmlutils.js

```diff
@@ -115,11 +115,11 @@ let htmlUtils = {
       return `
 <div class="switch-container ${action}">
   <div class="switch-toggle switch-3 switch-candy">
-    <input id="block-${origin_id}" name="${origin}" value="${constants.BLOCK}" type="radio" ${is_checked(constants.BLOCK, action)}>
+    <input id="block-${origin_id}" name="${origin}" value="${constants.BLOCK}" type="radio" aria-label="${i18n.getMessage("domain_slider_block_tooltip")}: ${origin}" ${is_checked(constants.BLOCK, action)}>
```

Comment:
Looking up messages using `chrome.i18n.getMessage()` is relatively slow and could slow down the page when there are hundreds of sliders to render.

See above in this function where we cache several translation strings in `tooltips`.